src/content/topics/home/advanced/relay-proxy/index.mdx

```diff
@@ -1,17 +1,17 @@
 ---
 path: '/home/advanced/relay-proxy'
 title: 'The Relay Proxy'
-description: 'This topic explains what the Relay Proxy is and how to use it. The Relay Proxy is an open-source microservice that connects to LaunchDarkly. The Relay Proxy lets servers connect to a local stream instead of making outbound connections to LaunchDarkly''s streaming service.'
+description: 'This topic explains what the Relay Proxy is and how to use it. The Relay Proxy is an open-source application that connects to LaunchDarkly. The Relay Proxy lets servers connect to a local stream instead of making outbound connections to LaunchDarkly''s streaming service.'
 published: true
 ---
 
 ## Overview
 
 This topic explains what the Relay Proxy is and how to use it.
 
-The Relay Proxy is a microservice that connects to the LaunchDarkly streaming API and proxies that connection to multiple clients.
+The Relay Proxy is a small Go application that connects to the LaunchDarkly streaming API and proxies that connection to clients within an organziation's network.
 
-The Relay Proxy lets a number of servers connect to a local stream instead of making a large number of outbound connections to LaunchDarkly's streaming service (`stream.launchdarkly.com`). You can configure the Relay Proxy to carry multiple environment streams from multiple projects.
+The Relay Proxy lets private servers connect to a local stream instead of making a large number of outbound connections to LaunchDarkly's streaming service (`stream.launchdarkly.com`). You can configure the Relay Proxy to carry multiple environment streams from multiple projects.
```

Comment:
Although it is most often the case that the servers are private, there's nothing stopping public servers or clients from connecting to Relay.

Reply:
Edited.  Is better?

Your point does kind of beg the question though, what does "local" mean here if public clients are connecting to it?

Reply:
Maybe this?

> The Relay Proxy lets client applications connect to a private stream instead of making a large number of outbound connections to LaunchDarkly's public streaming service